rust/ddlog_benches/benches/twitter.rs

```diff
@@ -14,8 +14,8 @@ fn record_counts() -> impl Iterator<Item = usize> {
     (50_000..=200_000).step_by(50_000)
 }
 
-fn twitter(c: &mut Criterion) {
-    let mut group = c.benchmark_group("twitter");
+fn twitter_micro(c: &mut Criterion) {
```

Comment:
You probably want to do this same thing to the live-journal benchmark

Reply:
I did.

PS. I rebased this branch locally and am still tweaking it.  Will force-push later today.